File: server/src/main/java/com/kulygin/dto/list/FeedbackListDto.java
package com.kulygin.dto.list;

import com.kulygin.dto.FeedbackDto;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class FeedbackListDto {
    
    private List<FeedbackDto> feedbacks;
    
}

File: client/src/main/web/src/app/dto/list/feedback-list-dto.ts
import {FeedbackDto} from '../feedback-dto';

export class FeedbackListDto {

    private _feedbacks: FeedbackDto[] = [];

    constructor(feedbacks: FeedbackDto[]) {
        this._feedbacks = feedbacks;
    }

    get feedbacks(): FeedbackDto[] {
        return this._feedbacks;
    }

    set feedbacks(value: FeedbackDto[]) {
        this._feedbacks = value;
    }

}